### ISSUE TITLE
RN App Screen is blank

### ISSUE THREAD

**Issue by ericmasiello** [ReactWindows/react-native-windows#614]
Hi,
I've tried walking through everything necessary to setup RN for Windows Universal Apps. When i run `react-native run-windows` the app starts up and the RN packager also kicks off but when the app actually starts up its just a blank white screen. Here's what I see from within the packager.

```
 ┌────────────────────────────────────────────────────────────────────────────┐ 
 │  Running packager on port 8081.                                            │ 
 │                                                                            │ 
 │  Keep this packager running while developing on any JS projects. Feel      │ 
 │  free to close this tab and run your own packager instance if you          │ 
 │  prefer.                                                                   │ 
 │                                                                            │ 
 │  https://github.com/facebook/react-native                                  │ 
 │                                                                            │ 
 └────────────────────────────────────────────────────────────────────────────┘ 
Looking for JS files in
   C:\Users\Eric\Documents\RNApp

[4:09:03 PM] <START> Building Dependency Graph
[4:09:03 PM] <START> Crawling File System
[Hot Module Replacement] Server listening on /hot

React packager ready.

[4:09:25 PM] <END>   Crawling File System (21781ms)
[4:09:25 PM] <START> Building in-memory fs for JavaScript
[4:09:25 PM] <END>   Building in-memory fs for JavaScript (141ms)
[4:09:25 PM] <START> Building in-memory fs for Assets
[4:09:25 PM] <END>   Building in-memory fs for Assets (109ms)
[4:09:25 PM] <START> Building Haste Map
[4:09:25 PM] <START> Building (deprecated) Asset Map
[4:09:25 PM] <END>   Building (deprecated) Asset Map (47ms)
[4:09:25 PM] <END>   Building Haste Map (125ms)
[4:09:25 PM] <END>   Building Dependency Graph (22281ms)
[4:09:35 PM] <START> request:/index.windows.bundle?platform=windows&dev=true&hot=false
[4:09:35 PM] <START> find dependencies
transformed 607/607 (100%)
[4:09:36 PM] <END>   find dependencies (1375ms)
[4:09:37 PM] <END>   request:/index.windows.bundle?platform=windows&dev=true&hot=false (1547ms)

```
I also tried just running the solution from within Visual Studio Community 2015 in debugger mode. However, when I do that, the app throws an uncaught exception and terminates. I haven't touched or modified any of the vanilla code after running the setup steps.

Also, I tried temporarily completely turning off my firewall as I heard some people run into issues with the node communicating with the app. Still no luck.

I'm running this is a fully updated Windows 10 inside of a VM running on VMWare Fusion. Not sure if that would make any difference or not but I figured its worth mentioning. Any suggestions?

**Comment by rozele** [comment]
@ericmasiello Which version of Windows 10 SDK are you using?

**Comment by ericmasiello** [comment]
@rozele looks like 10586 to me. Is this where I verify?
<img width="547" alt="screen shot 2016-08-22 at 10 08 55 pm" src="https://cloud.githubusercontent.com/assets/3525886/17877928/1889472a-68b5-11e6-876a-5509088db451.png">

**Comment by ericmasiello** [comment]
@rozele looks like 10586 to me. Is this where I verify?

<img width="548" alt="screen shot 2016-08-22 at 10 13 44 pm" src="https://cloud.githubusercontent.com/assets/3525886/17878019/c9028bd4-68b5-11e6-870d-d96a58d7ee2f.png">

Oddly, I noticed that if I change the min-version also to 10586, and then run `react-native run-windows` I get this error:

```
PS C:\Users\Eric\Documents\RNApp> react-native run-windows
Restoring NuGet packages
MSBuild auto-detection: using msbuild version '14.0' from 'C:\Program Files\MSBuild\14.0\bin'.
Restoring packages for C:\Users\Eric\Documents\RNApp\windows\RNApp\project.json...
Committing restore...
C:\Users\Eric\Documents\RNApp\windows\RNApp\RNApp.csproj
Restore completed in 2113ms.
Restoring packages for C:\Users\Eric\Documents\RNApp\node_modules\react-native-windows\ReactWindows\ReactNative\project.json...
Committing restore...
C:\Users\Eric\Documents\RNApp\node_modules\react-native-windows\ReactWindows\ReactNative\ReactNative.csproj
Restore completed in 978ms.

NuGet Config files used:
    C:\Users\Eric\AppData\Roaming\NuGet\NuGet.Config
    C:\ProgramData\nuget\Config\Microsoft.VisualStudio.Offline.config

Feeds used:
    https://api.nuget.org/v3/index.json
    C:\Program Files\Microsoft SDKs\NuGetPackages\

Building C:/Users/Eric/Documents/RNApp/windows/RNApp.sln
ERROR: The system was unable to find the specified registry key or value.
Found MSBuild v14.0 at C:\Program Files\MSBuild\14.0\bin\
ERROR: The system was unable to find the specified registry key or value.
Found MSBuild v4.0 at C:\Windows\Microsoft.NET\Framework\v4.0.30319\
Building Solution: C:/Users/Eric/Documents/RNApp/windows/RNApp.sln
Build configuration: Debug
Build platform: x86
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  ReactNative -> C:\Users\Eric\Documents\RNApp\node_modules\react-native-windows\ReactWindows\ReactNative\bin\x86\Debug\ReactNative.dll
  RNApp -> C:\Users\Eric\Documents\RNApp\windows\RNApp\bin\x86\Debug\RNApp.exe
  RNApp -> C:\Users\Eric\Documents\RNApp\windows\RNApp\AppPackages\RNApp_1.0.0.0_x86_Debug_Test\RNApp_1.0.0.0_x86_Debug.appx
  RNApp -> C:\Users\Eric\Documents\RNApp\windows\RNApp\AppPackages\RNApp_1.0.0.0_x86_Debug_Test\RNApp_1.0.0.0_x86_Debug.appxsym

React-Native Server already started
Removing old version of the app
Installing new version of the app
Failed to deploy: Command failed: CheckNetIsolation LoopbackExempt -a -n=
```

I'm not sure if any of that is telling or not. 

Otherwise, if i change min version back to whats in the screen shot, it reverts to the behavior I mentioned previously.

**Comment by rozele** [comment]
@ericmasiello is this for the default app that is generated from `react-native windows` or do you have some custom logic?  If it's custom logic, can you share a repro?

**Comment by ericmasiello** [comment]
@rozele no this is the default app you get with `react-native windows`. I haven't changed anything.

**Comment by ericmasiello** [comment]
@rozele Do I need to be on a 64 bit version of Windows 10? I'm currently running a 32 bit version

**Comment by rozele** [comment]
@ericmasiello there shouldn't be any issue with architecture (XAML should behave correctly no matter the underlying architecture), although I haven't tried running on a 32-bit version.

**Comment by ericmasiello** [comment]
Alright. I'm gonna close this. I don't have any explanation but after spending the day setting up a new Windows 10 32 bit VM, I was able to get the project running. The only thing I'll call out is the in the README, it has a link that says where to get the Windows 10 SDK. That link didn't seem to get me the latest SDK. Instead, after running `react-native windows`, I had to open up the sln file. That then prompted me to download the Windows 10 SDK version mentioned in the README. Other than that, everything seems to be working now. Thanks for your help @rozele
